batteries.opam

```diff
@@ -23,10 +23,14 @@ depends: [
   "ocamlfind" {build & >= "1.5.3"}
   "ocamlbuild" {build}
   "qtest" {with-test & >= "2.5"}
-  "qcheck" {with-test & >= "0.6" & < "0.14"}
+  "qcheck" {with-test & >= "0.9" & < "0.14"}
   "benchmark" {with-test & >= "1.6"}
   "num"
 ]
+conflicts: [
+  "base-effects"
+  "ocaml-option-no-flat-float-array"
+]
 # url {
 #   src: "https://github.com/ocaml-batteries-team/batteries-included/archive/vXXX.tar.gz"
 #   checksum: "md5=YYY"
```

Comment:
```suggestion
```
Also this is a bit unnecessary and surprising for users looking for examples. Using `opam publish` is much easier to do this job rather than doing it by hand.

Reply:
you can change md5 to something else, if that's your concern